source/features/parse-backticks.tsx

```diff
@@ -41,6 +43,14 @@ function init(): void {
 			parseBackticks(element);
 		}
 	});
+
+	// `isRepoSearch`
```

Comment:
Perhaps it needs to specify _why_ this needs its own version
```suggestion
	// `isRepoSearch` might highlight keywords inside backticks, breaking the regular dom-formatter #3509
```